src/platform/lumin-runtime/elements/builders/page-view-builder.js

```diff
@@ -5,94 +5,95 @@ import { Alignment } from '../../types/alignment';
 
 import { UiNodeBuilder } from './ui-node-builder.js';
 import { ArrayProperty } from '../properties/array-property.js';
+import { ClassProperty } from '../properties/class-property.js';
```

Comment:
why are you importing this if you're not using it?

Reply:
I am using it: line 27. However, github displays the updates for those files in very messed up way.